pkg/pvc-autoresizer/pvc_informerV2.go

```diff
@@ -0,0 +1,166 @@
+package autoresizer
+
+import (
+	"context"
+
+	hwameistorv1alpha1 "github.com/hwameistor/hwameistor/pkg/apis/hwameistor/v1alpha1"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type pvcWorker struct {
+	Namespace *corev1.Namespace
+	StorageClass *storagev1.StorageClass
+	cli client.Client
+}
+
+func newPVCWorker(cli client.Client) *pvcWorker {
+	return &pvcWorker{
+		cli: cli,
+	}
+}
+
+func (w *pvcWorker) conformPVCAgainstResizePolicy(pvc *corev1.PersistentVolumeClaim, rp *hwameistorv1alpha1.ResizePolicy) (bool, error) {
+	if rp.Spec.PVCSelector != nil {
+		matched, err := matchPVC(rp, pvc)
+		if err != nil {
+			log.Errorf("match pvc err: %v", err)
+			return false, err
+		}
+		if !matched {
```

Comment:
why return false? should it continue to be checked more?

Reply:
it's && relation between selectors of a resizepolicy, so if one selector doesn't match, the whole resizepolicy doesn't match.